### ISSUE TITLE
Remove unused code.

### ISSUE THREAD

**Issue by a-t-0** [HiveMinds-EU/Taskwarrior-installation#73]

**Comment by mohitsaxenaknoldus** [comment]
In which files is this unused code?

**Comment by a-t-0** [comment]
At least in: 
https://github.com/HiveMinds-EU/Taskwarrior-installation/tree/master/src/examples
And in the accompanying tests:
https://github.com/HiveMinds-EU/Taskwarrior-installation/tree/master/test

If you detect any additional unused code, feel free to remove it.

**Comment by mohitsaxenaknoldus** [comment]
Ok, I'll do it. Please assign it to me.

**Comment by a-t-0** [comment]
> Ok, I'll do it. Please assign it to me.

Awesome! Thank you! For my completeness, were you not able to assign yourself, or did you consider this better protocol? (Otherwise I might have to change settings to enable people to assign themselves.)

**Comment by mohitsaxenaknoldus** [comment]
> > Ok, I'll do it. Please assign it to me.
> 
> Awesome! Thank you! For my completeness, were you not able to assign yourself, or did you consider this better protocol? (Otherwise I might have to change settings to enable people to assign themselves.)

I couldn't assign myself :(

**Comment by mohitsaxenaknoldus** [comment]
PR opened: https://github.com/HiveMinds-EU/Taskwarrior-installation/pull/77
Please review @a-t-0

**Comment by a-t-0** [comment]
Thank you @mohitsaxenaknoldus ! I have merged your pull request into the main branch, and invited you as collaborator with Triage-capabilities, I hope that enables you to assign yourself to issues if you feel like picking up a challenge! (Feel free to arbitraily test it on some issue (regardless of whether you will work on it or not), and letting me know whether you were indeed able to assign your self (in a comment to that same issue), I would appreciate the feedback!

Also, I took the liberty to remove additional files in the src directory that were not used, removed all the test files and the example files. They are already contained in this repository (for reference) https://github.com/a-t-0/shell_unit_testing_template/tree/main/test and are not used in this Taskwarrior installer. However, this required quite some insight in how this entire code works, so I should have been more clear on that, or explicitly have specified what exactly the intentions were in this issue. Thanks for making that clear to me!
